test/cpp/cpp_test_util.h

```diff
@@ -9,7 +9,7 @@
 #include <unordered_set>
 
 #include "absl/types/span.h"
-#include "tensorflow/compiler/xla/xla_client/computation_client.h"
+#include "third_party/xla_client/computation_client.h"
```

Comment:
since we don't need to move `xla_client` into tf to build anymore, maybe we can move the whole `xla_client` under the `torch_xla/csrc/`? We can do that in the next pr.

Reply:
oh nvm, I think xla_client has its own build file and uses bazel. It is more than just copying file over.

Reply:
This is the first (easier) step; migrating the libxla_c_c.so to bazel, since it's already in bazel. When we migrate the plugin to bazel then we can merge the two builds, that will give us more cache, faster build times, faster incremental builds, etc.

We can consider doing like jax does in the future, and splitting into jax/jaxlib equivalents where one is python+thin layer of cc binidngs, and the other is the cc library, mostly for ease of pushes where you'd change one more often than the other. We can discuss that after the migration and see what makes most sense.